CoreScriptsRoot/Modules/Settings/Utility.lua

```diff
@@ -1698,7 +1698,7 @@ local function CreateNewSlider(numOfSteps, startStep, minStep)
 
 	local function mouseUpFunc(inputObject)
 		if not interactable then return end
-		if not isActivateEvent() then return end
+		if not isActivateEvent(inputObject) then return end
```

Comment:
has this fix been testing with all types of input? keyboard, mouse, touch and controller?